.github/workflows/branches.yml

```diff
@@ -12,56 +12,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - id: cache-packages-volumes
-        uses: actions/cache@v2
-        with:
-          path: |
-            /tmp/packages-volumes.tar
-          key: packages-{{ hashFiles('**/package-lock.json') }}-{{ hashFiles('**/Gemfile.lock') }}
-      - id: cache-docker
-        uses: actions/cache@v2
-        with:
-          path: /tmp/docker-registry
-          key: dr-etna-{{ hashFiles('**/docker/**') }}-{{ hashFiles('**/Dockerfile.development') }}-{{ hashFiles('**/docker-compose.yml') }}
-      - name: Ready images from cache
-        run: |
-          docker run -d -p 5000:5000 --restart=always --name registry -v /tmp/docker-registry:/var/lib/registry registry:2 && npx wait-on tcp:5000
-          REPOSITORY=localhost:5000\\/mountetna make tag
-          cd docker && docker-compose -p monoetna pull || true
-      - name: Ready volumes and containers
-        run: |
-          cd docker && make create
-      - run: cd docker && docker-compose -p monoetna push || true
-        if: steps.cache-docker.outputs.cache-hit != 'true'
-        name: Push images to cache location
-      - name: Load volumes from cache
-        run: |
-          docker run --rm -v /tmp:/backup/ `docker volume list -q | egrep -v '^.{64}$' | egrep 'node-modules|gems' | awk '{print "-v " $1 ":/mnt/" $1}'` alpine tar -xf /backup/packages-volumes.tar -C /mnt
-        if: steps.cache-packages-volumes.outputs.cache-hit == 'true'
       - name: Run metis test suite
         id: metis_first_test_run
+        env: &release-env
```

Comment:
This will instruct the `build_image` and related build scripts to build / cache from a `master` image in the dockerhub `etnaagent` repo which I have established (free, etna-agent@ucsf.edu email).